packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -51,11 +53,41 @@ function useBlockEditorSettings( settings, hasTemplate ) {
 				canUser( 'create', 'media' ),
 				true
 			),
+			baseUrl: getUnstableBase()?.url || '',
 		};
 	}, [] );
 
 	const { undo } = useDispatch( editorStore );
 
+	// Temporary home - should this live in `core-data`?
+	function fetchRichUrlData( url, fetchOptions = {} ) {
+		const emptyDataSet = {};
```

Comment:
If we want to create a constant empty object, then we should probably declare it outside the render function, like in the top level, so that the instance never changes.